model/src/pyrenew/deterministic/nullrv.py

```diff
@@ -139,8 +139,8 @@ def sample(
 
         Parameters
         ----------
-        predicted : ArrayLike
-            Rate parameter of the Poisson distribution.
+        mu : ArrayLike
+            Rate parameter of the Poisson distribution. #TODO
```

Comment:
I enjoy, in personal projects, using `TODO`s in docstrings. Dylan has strong opinions against the use of TODOs in most places save for GitHub issues. I agree with this, though do not feel as strongly as Dylan does. The `TODO` here and the norm of having them is in the MSR code is your call as project lead, however it might be useful for everyone to exchange several sentences on the value of including `TODO`s in code.

Reply:
Forgot about this. Thanks for catching.